Core/src/org/sleuthkit/autopsy/modules/tester/TestFactory.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package org.sleuthkit.autopsy.modules.tester;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+import org.openide.util.Exceptions;
+import org.openide.util.lookup.ServiceProvider;
+import org.sleuthkit.autopsy.coreutils.ImageUtils;
+import org.sleuthkit.autopsy.coreutils.VideoUtils;
+import org.sleuthkit.autopsy.ingest.DataSourceIngestModule;
+import org.sleuthkit.autopsy.ingest.DataSourceIngestModuleProgress;
+import org.sleuthkit.autopsy.ingest.IngestModuleFactory;
+import org.sleuthkit.autopsy.ingest.IngestModuleIngestJobSettings;
+import org.sleuthkit.datamodel.Content;
+
+@ServiceProvider(service = IngestModuleFactory.class)
+public class TestFactory  implements IngestModuleFactory {
+
+    @Override
+    public String getModuleDisplayName() {
+        return "Kellys Test Module";
+    }
+
+    @Override
+    public String getModuleDescription() {
+        return "A happy fun time.";
+    }
+
+    @Override
+    public String getModuleVersionNumber() {
+        return "1";
+    }
+    
+    @Override
+    public boolean isDataSourceIngestModuleFactory() {
+        return true;
+    }
+    
+    @Override
+    public DataSourceIngestModule createDataSourceIngestModule(IngestModuleIngestJobSettings settings) {
+        return new TestModule();
+    }
+    
+    class TestModule implements DataSourceIngestModule {
+
+        @Override
+        public ProcessResult process(Content dataSource, DataSourceIngestModuleProgress progressBar) {
+            File outputFile = new File("C:\\Temp\\output\\output.avi");
```

Comment:
I'm not sure I understand what are these paths and how is this class used. Looks like some leftover test code perhaps? If so, I will use it for my testing but then we should remove it.

Reply:
@eugene7646 I thought I was being careful, this code should not have come into the PR.  I will remove it.